.test-infra/jenkins/LoadTestConfig.groovy

```diff
@@ -0,0 +1,621 @@
+            private def _keySize
+            private def _valueSize
+            private def _numHotKeys
+            private def _hotKeyFraction
+
+            //internal usage
+            private SDK i_sdk
+
+            private InputOptions() {}
+
+            static withSDK(final SDK sdk) {
+                final def input = new InputOptions()
+                input.i_sdk = sdk
+                return input
+            }
+
+            void numRecords(final int num) { _numRecords = num }
+            void keySize(final int size) { _keySize = size }
+            void valueSize(final int size) { _valueSize = size }
+            void numHotsKeys(final int num) { _numHotKeys = num }
+            void hotKeyFraction(final int fraction) { _hotKeyFraction = fraction }
+
+            @Override
+            String toPrimitiveValues() {
+                return '\'' + new JsonBuilder(ConfigHelper.convertProperties(propertiesMap, i_sdk)).toString() + '\''
+            }
+
+            static InputOptions of(final InputOptions oldOptions) {
+                final def newOptions = new InputOptions()
+                InvokerHelper.setProperties(newOptions, oldOptions.propertiesMap)
+                return newOptions
+            }
+
+            LinkedHashMap<String, Object> getPropertiesMap() {
+                return [
+                        i_sdk: i_sdk,
+                        _numRecords: _numRecords,
+                        _keySize: _keySize,
+                        _valueSize: _valueSize,
+                        _numHotKeys: _numHotKeys,
+                        _hotKeyFraction: _hotKeyFraction
+                ] as LinkedHashMap<String, Object>
+            }
+        }
+
+        private static class SourceOptions implements SerializableOption<String> {
+            private def _numRecords
+            private def _keySizeBytes
+            private def _valueSizeBytes
+            private def _numHotKeys
+            private def _hotKeyFraction
+            private def _splitPointFrequencyRecords
+
+            //internal usage
+            private SDK i_sdk
+
+            private SourceOptions() {}
+
+            static withSDK(final SDK sdk) {
+                final def input = new SourceOptions()
+                input.i_sdk = sdk
+                return input
+            }
+
+            void numRecords(final int num) { _numRecords = num }
+            void keySizeBytes(final int size) { _keySizeBytes = size }
+            void valueSizeBytes(final int size) { _valueSizeBytes = size }
+            void numHotsKeys(final int num) { _numHotKeys = num }
+            void hotKeyFraction(final int fraction) { _hotKeyFraction = fraction }
+            void splitPointFrequencyRecords(final int splitPoint) { _splitPointFrequencyRecords = splitPoint }
+
+            @Override
+            String toPrimitiveValues() {
+                return '\'' + new JsonBuilder(ConfigHelper.convertProperties(propertiesMap, i_sdk)).toString() + '\''
```

Comment:
What do you think about changing it into notation "'${...}'"?

Reply:
Thanks, will change it!